src/commcare_cloud/ansible/roles/postgresql/tasks/main.yml

```diff
@@ -33,7 +33,7 @@
   tags: pghashlib
 
 - name: Setup plproxy
-  include: plproxy.yml
+  include_tasks: plproxy.yml
   when: item.get('host')
   loop:
     - "{{ postgresql_dbs.form_processing.proxy|default({}) }}"
```

Comment:
the tasks are dynamic because their inclusion depends on the presence of the host attribute in each item of the loop.